lib/index.js

```diff
@@ -33,13 +89,18 @@ function start() {
   }
   const port = findPort();
   const p = spawn(process.execPath, [require.resolve('./worker'), port, ...FLAGS], {
-    stdio: 'inherit',
+    stdio: debugMode ? ['ignore', outputFile, errorFile] : 'inherit',
     windowsHide: true,
   });
   p.unref();
   process.on('exit', () => {
     p.kill();
   });
+  client = net.connect(port, '127.0.0.1', () => {
+    // fs.appendFileSync(path.join(responseFolder, 'clientConnection.txt'), 'connected');
+    console.log('client connected.');
+    clientConnected = true;
+  });
   waitForAlive(port, 1);
```

Comment:
?

Reply:
It is original logic for checking connection success.